manifests/repo.pp

```diff
@@ -1,4 +1,8 @@
 # @summary Handles repositories for the container runtime
+#
+# @param manage_container_manager whether to add cri-o repository or not
```

Comment:
Does this parameter only manage the repo? From the name it seems as if it will do more.

Reply:
in this particular class it only manages the presence of the repo. there is another parameter of the same name in the main class which would do more.